Doppler.PushContact/Services/Messages/IMessageSender.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Doppler.PushContact.Services.Messages
+{
+    public interface IMessageSender
+    {
+        Task<IEnumerable<SendMessageTargetResult>> SendAsync(string title, string body, IEnumerable<string> targetDeviceTokens);
```

Comment:
No estoy seguro, pero me suena a que sería más simple y flexible que un objeto contenga las respuestas de cada envío.

```suggestion
        Task<SendMessageResult> SendAsync(string title, string body, IEnumerable<string> targetDeviceTokens);
```

Reply:
Hecho!